plugins/module_utils/network/eos/argspec/ospf_interfaces/ospf_interfaces.py

```diff
@@ -109,9 +109,9 @@ def __init__(self, **kwargs):
                                         "null",
                                     ],
                                 },
-                                "keytype": {"type": "str"},
+                                "keytype": {"type": "str", "no_log": True},
```

Comment:
Does keytype sensitive data? If not then we can set no_log: False explicitly to bypass validate-modules.